src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -151,10 +165,33 @@ CHIP_ERROR BLEManagerImpl::_Init()
 
     memset(mSubscribedConns, 0, sizeof(mSubscribedConns));
 
-    ReturnErrorOnFailure(InitRandomStaticAddress());
-    int err = bt_enable(NULL);
+#ifdef CONFIG_BT_BONDABLE
+    static_assert(CONFIG_BT_ID_MAX == 2);
```

Comment:
Is that actually necessary? Users might add more IDs and that's fine, right?

Reply:
The problem with bigger number of ids than 1 (no bonding) or 2 (bonding + Matter) is that it's hard to know which one of them should be used to what purpose after loading it from settings. For id 0 it obvious (it's default), but if user created one more beside the Matter's one, we are going to load ids 1 and 2. The order of creation will decide about the id value and I was afraid we will not be able to figure it out.

Nevertheless we create Matter id in the `Init` method, so maybe it's safe to assume it will be always equal to 1, and the user's ids will be created using following numbers.

Reply:
Yes, I think that given that you call this just after `bt_enable`, it will always be 1.

Reply:
Ok, applied.